integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/ExistsPredicateSpecificsIT.java

```diff
@@ -92,7 +92,7 @@ public void missing() {
 		String fieldPath = mainIndex.binding().fieldWithDefaults.get( dataSet.fieldType ).relativeFieldName;
 
 		assertThatQuery( mainIndex.query()
-				.where( f -> f.bool().mustNot( f.exists().field( fieldPath ) ) )
+				.where( f -> f.not( f.exists().field( fieldPath ) ) )
```

Comment:
Could you please have look at the documentation too? I see [here](https://docs.jboss.org/hibernate/search/6.2/reference/en-US/html_single/#search-dsl-predicate-exists) we mention `mustNot`, but `not` would be more appropriate (perhaps with a link to the relevant section).